i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,45 @@
+from torch import nn
+
+
+class ConformerFeedForwardv1(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(self, input_dim, hidden_dim, dropout=0.1):
+        """
+        :param int input_dim: input dimension
+        :param int hidden_dim: hidden dimension (normally set to 4*input_dim as suggested by the paper)
+        :param float dropout: dropout probability
+        """
+        super().__init__()
+
+        self.layer_norm = nn.LayerNorm(normalized_shape=input_dim)
+
+        self.linear_layer_1 = nn.Linear(
+            in_features=input_dim, out_features=hidden_dim, bias=True
+        )
+
+        self.swish_activation = nn.SiLU()
```

Comment:
This was configurable in our earlier implementation. So I would just call it `activation`. And I would make it configurable.